### ISSUE TITLE
about report

### ISSUE THREAD

**Issue by jasperchang504** [2020-DL-Training-Program/Lab1-Backpropagation#8]
助教
Experiment setups (30%):
A. Sigmoid functions
B. Neural network
C. Backpropagation
這個是要寫怎麼實現這些function嗎?

**Comment by zivzone** [comment]
@jasperchang504 是的

**Comment by andystrike** [comment]
請問助教，在Lab的pdf有提到: In the training, you need to print loss.
我們可以將"Loss對epoch的變化來作圖"取代這項嗎?

**Comment by zivzone** [comment]
@andystrike OK

**Comment by jasperchang504** [comment]
助教請問一下 我們第二個Experiment setups (30%)是要貼code解釋嗎

**Comment by zivzone** [comment]
@jasperchang504 主要是希望你們解釋如何實作出這個功能，建議附上 code 這樣應該比較好寫。但如果不附上 code 也可以詳細且清楚說明自己是如何實作出的也 ok。
